modules/ROOT/content-nav.adoc

```diff
@@ -178,6 +177,7 @@
 ** Integration with auth systems
 *** xref:authentication-authorization/ldap-integration.adoc[]
 *** xref:authentication-authorization/sso-integration.adoc[]
+** xref:authentication-authorization/password-and-user-recovery.adoc[]
```

Comment:
I think we should experiment a bit and see if this page can be incorporated into the **Manage users** page, or else we could add it after ** xref:authentication-authorization/manage-roles.adoc[].

Reply:
I think it's a good idea to incorporate the content to the Manage users page. Should I commit this change here or open another PR?

Reply:
Hmm, let's keep it in one PR so that we see what we are doing.